<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Commodity</class>
 <widget class="QWidget" name="Commodity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="CommodityName">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#CommodityName
{
	color:rgb(54,54,54);
	font-size: 16;
	font-family: &quot;微软雅黑&quot;;
}</string>
   </property>
   <property name="text">
    <string>CommodityName</string>
   </property>
  </widget>
  <widget class="QLabel" name="CostText">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#CostText	
{

	font: 14pt &quot;微软雅黑&quot;;
}</string>
   </property>
   <property name="text">
    <string>CostText</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>611</x>
     <y>360</y>
     <width>29</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="minusBtn">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>360</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="plusBtn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>360</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBuy">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>400</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>直接购买</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ptnBuy">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>400</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>加入购物车</string>
   </property>
  </widget>
  <widget class="QWidget" name="widUPic" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-1</y>
     <width>491</width>
     <height>491</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>160</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>256</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>160</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>140</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
   <property name="text">
    <string>编号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>140</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
   <property name="text">
    <string>类型：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
   <property name="text">
    <string>价格:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>0</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
   <property name="text">
    <string>名称:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
